lib/common/MongoDbTransactionStore.ts

```diff
@@ -19,7 +20,9 @@ export default class MongoDbTransactionStore implements ITransactionStore {
    * Initialize the MongoDB transaction store.
    */
   public async initialize (serverUrl: string, databaseName: string): Promise<void> {
-    const client = await MongoClient.connect(serverUrl, { useNewUrlParser: true }); // `useNewUrlParser` addresses nodejs's URL parser deprecation warning.
+    // `useNewUrlParser` addresses nodejs's URL parser deprecation warning.
+    const client = await MongoClient.connect(serverUrl, { useNewUrlParser: true, logger: MongoDbLogger.customLogger });
```

Comment:
Any reason why this construction doesn't do the below like others:
```
      monitorCommands: true,
      loggerLevel: 'debug'
```

Reply:
Fixed after refactoring